packages/xo-web/src/common/tags.js

```diff
@@ -85,6 +108,26 @@ export default class Tags extends Component {
     event.preventDefault()
   }
 
+  _selectExistingTags = async () => {
+    await confirm({
+      body: <SelectExistingTag />,
+      icon: 'add',
+      title: _('selectExistingTags'),
+    }).then(tags => Promise.all(tags.map(this._addTag)), noop)
+    this._stopEdit()
```

Comment:
IMO this doesn't need to wait for the tags to be added in the server. Also, avoid mixing `.then`s and `await`s as it can get confusing.